script.js

```diff
@@ -0,0 +1,90 @@
+let selectMenu = document.querySelector("#products");
+let category = document.querySelector(".right h2");
+let container = document.querySelector(".product-wrapper");
+
+selectMenu.addEventListener("change", function(){
+	let categoryName = this.value;
+	category.innerHTML = this[this.selectedIndex].text;  
+
+	let http = new XMLHttpRequest();
+	http.onreadystatechange = function(){
+		if(this.readyState == 4 && this.status == 200){
+			let response = JSON.parse(this.responseText);
+			// let response = this.responseText;
+			let out = "";
+			for(let item of response){
+				out += `
+					<div class="product">
+						<div class="left">
+							<img src="images/placeholder.png" alt="">
+						</div>
+						<div class="right">
+							<p class="title">${item.postingTitle}</p>
+							<p class="price">${item.price}&euro;</p>
+							<p class="description">${item.location}</p>
+						</div>
+					</div>
+				`;
+			}
+			container.innerHTML = out;
+		};
+	}	
+	http.open('POST', "script.php", true);
```

Comment:
Maybe a more descriptive name than `script.php` -- the name of files you call via AJAX are kinda like function names, so it's good if they're descriptive